scalableminds/webknossos
# Fix team selection in advanced dataset list view

To be honest I don't really understand why this commit fixes the issue but it does. We're using the class property initializer syntax elsewhere for Backbone properties as well and there it works.

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Fix team selection in advanced dataset list view.

### Steps to test:
- Open the Dashboard and select the advanced dataset view. Click on one of the team labels for one dataset, a list of teams should appear (this didn't happen before the fix).
- Select some teams and save.

### Issues:
- fixes #1884

------
